test/1.4/expressions/T_endian_int.dml

```diff
@@ -183,11 +181,15 @@ method test() throws {
 
     // Trait method call with eint arg
     // indirect, throws
-    tp.s_m_t(var);
+    try {
+        tp.s_m_t(var);
+    } catch {}
```

Comment:
Rather than wrapping in try/catch, just remove the throws from s_m_t

Reply:
This was my first thought too, but `s_m_t` throwing is intentional and part of the test -- that's why it has the `_t` suffix.

Reply:
you are right. In cases like this I suppose the empty try/catch is fine. If the other cases I've complained about are similar then disregard all of it

Reply:
Still makes sense to extend the catch block with an `assert false`, for clarity.